### ISSUE TITLE
Cannot Bind FlowItemSource

### ISSUE THREAD

**Issue by Willjobs94** [daniel-luberda/DLToolkit.Forms.Controls#32]
I have nuget package version 2.0.0
When I set the Items property on my view model  and navigate to the page that contains the flowlistview, it does not work, and even freeze the application.

If I remove the binding the page load.

**Comment by daniel-luberda** [comment]
Do you have any code? Without it I can't help you. FlowListView works fine (just test example project).

**Comment by Willjobs94** [comment]
This is my list view
			```
                             <flv:FlowListView
				x:Name="listview"
				FlowColumnCount="1"
				SeparatorVisibility="None"
				HasUnevenRows="false"
				FlowItemsSource="{Binding Items}">
				<flv:FlowListView.FlowColumnTemplate>
					<DataTemplate>
						<Label
							HorizontalOptions="Fill"
							VerticalOptions="Fill"
							XAlign="Center"
							YAlign="Center"
							Text="Test" />
					</DataTemplate>
				</flv:FlowListView.FlowColumnTemplate>
			</flv:FlowListView>```

and I'm setting the items in my ViewModel constructor like this.
			`Items = new ObservableCollection<object>
			{
				new { }
			};`
If I leave Items null the page load, if a put any object in the Items collection the page does not load. I've already cloned the project and check the example, but I didn't find the reason why mine is not working.

**Comment by daniel-luberda** [comment]
@Willjobs94 Works just fine for me. See this: 

![simulator screen shot 27 10 2016 03 00 03](https://cloud.githubusercontent.com/assets/13569983/19750458/830b32ec-9bf1-11e6-813b-12c3e83ae838.png)

**Comment by Willjobs94** [comment]
Thats the thing, It works on the sample project, but I don't find what is missing on my project, that it does not work. I does not throw any exception, it just freeze my app.

**Comment by daniel-luberda** [comment]
If you use it `XAML` only, do you call `FlowListView.Init()` somewhere as stated here: https://github.com/daniel-luberda/DLToolkit.Forms.Controls/tree/master/FlowListView#faq

**Comment by Willjobs94** [comment]
Somewhere where ?, I tried putting on my code-behind page, and on my platform specific project, if I use the view without setting the item property I can navigate to the page, without error. the app stuck if I set the Items property and navigate to that page., I'm actually using a TappedPage and don't know if that is the problem.

**Comment by daniel-luberda** [comment]
Could you attach some sample project so I could check what's wrong? It's hard to predict.

**Comment by Willjobs94** [comment]
This archive contains the view that have the problem, and where I'm using `FlowListView.Init()`
[Archive.zip](https://github.com/daniel-luberda/DLToolkit.Forms.Controls/files/554771/Archive.zip)

**Comment by daniel-luberda** [comment]
You shouldn't put any lists inside `ScrollView` - that may be the reason.

**Comment by Willjobs94** [comment]
I tried removing it, but that's not the problem.

**Comment by daniel-luberda** [comment]
It's not a full solution, it can be anything. Does it work with a normal `ListView`?

**Comment by Willjobs94** [comment]
Yes

**Comment by daniel-luberda** [comment]
- Do you use XAML compilation?
- Could you make a full repro with it?
- Could you try to use a class instead object? Eg: 

```C#
			Items = new ObservableCollection<ItemModel>
			{
				new ItemModel(),
				new ItemModel(),
				new ItemModel(),
			};
```

**Comment by Willjobs94** [comment]
* I do not use XAML compilation 
* I can't make a public repo with the source code
* I already tried with class instance and got the same result.

**Comment by daniel-luberda** [comment]
- Please try to enable XAML compilation, maybe it will you show some compilation errors.
- The one page with the same view and model (prism, etc) would be just fine, so I can just test it. It may be something prism specific, are you sure you implemented `INotifyPropertyChanged` correctly? I can't see any calls or do you use Fody?

**Comment by Willjobs94** [comment]
I use Fody, but I even tried implementing it with Xamvvm like in the project samples and still not working. Let me try with XAML compilation and see what I get. Thanks for the fast feedback

**Comment by Willjobs94** [comment]
I tried XAMLC but it didn't work, and not exception or error found.

**Comment by daniel-luberda** [comment]
That's strange, I used this control with different MVVM frameworks and didn't have any problems yet. I am wondering what could be the cause of it. I pasted the code and it just worked fine in another solution.

**Comment by Willjobs94** [comment]
Ok, let me try to use prism, on the sample project and see if that is the problem.

**Comment by daniel-luberda** [comment]
Thanks. If you have the sample project, feel free to mail it / send it to me. I'll try to debug it.

**Comment by Willjobs94** [comment]
I installed Prism on the sample project and works great, but it did not work on my project. After that I put the same page and view model on both project (sample project and my project) and still not working, then I saw that I had a Xamarin.Forms update, so I just updated an cleaned my solution and now is working in my app. I didn't have to use  `FlowListView.Init()` or XAML compiler, even thought I'll probably use the XAMLC at the end for optimization purpose. 

Thank you very much for the feedback, I think I can close this issue now.
